client/layout/store-notices/index.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Component } from '@wordpress/element';
+import { compose } from '@wordpress/compose';
+import PropTypes from 'prop-types';
+import { Notice } from '@wordpress/components';
+import { withSelect } from '@wordpress/data';
+
+/**
+ * WooCommerce dependencies
+ */
+import { OPTIONS_STORE_NAME } from '@woocommerce/data';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+class StoreWarnings extends Component {
+	isVisibleMissingPaimentWarning() {
+		const { paymentMethods, queryString } = this.props;
```

Comment:
Could you do this in the `withSelect` below, so that you do all the logic in one place and just get a `showMissingPaymentMethod` prop? That would make it much easier to refactor when another warning gets added.

Reply:
Good point! I moved that logic in the commit 44ed4c7